docs/index.html

```diff
@@ -180,11 +197,26 @@
 
 <body>
 <h2>#TwitterThrowback</h2>
-See what you've tweeted on this day in the past.
+<span>See what you've tweeted on <span id="this_day"></span> in the past.</span>
```

Comment:
```suggestion
<span>See what you've tweeted on <span id="this_day">this day</span> in the past.</span>
```

Better to have a good default, in case JS takes a while to process.